README.md

```diff
@@ -70,9 +70,45 @@ Main features of diktat are the following:
 To **autofix** all code style violations use `-F` option.
 
 ## Run with Maven
+### Use the diktat-maven-plugin
+This plugin is available since version 0.1.3. You can see how it is configured in our project for self-checks: [pom.xml](pom.xml).
+If you use it and encounter any problems, feel free to open issues on [github](https://github.com/cqfn/diktat/issues).
 
-### Use maven-antrun-plugin
+Add this plugin to your pom.xml:
+<details>
+  <summary><b>Maven plugin snippet</b></summary><br>
 
+```xml
+            <plugin>
+                <groupId>org.cqfn.diktat</groupId>
+                <artifactId>diktat-maven-plugin</artifactId>
+                <version>${diktat.version}</version>
+                <executions>
+                    <execution>
+                        <id>diktat</id>
+                        <phase>none</phase>
+                        <goals>
+                            <goal>check</goal>
+                            <goal>fix</goal>
+                        </goals>
+                        <configuration>
+                            <inputs>
+                                <input>${project.basedir}/src/main/kotlin</input>
+                                <input>${project.basedir}/src/test/kotlin</input>
+                            </inputs>
+                            <diktatConfigFile>diktat-analysis.yml</diktatConfigFile>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+```
+
+</details>
+
+To run diktat check use command `$ mvn diktat:check@diktat`.
```

Comment:
btw - always wanted to ask why we need `diktat` word twice: `diktat:` and `@diktat`

Reply:
We have plugin execution which is not bound to any phase to not waste time on it in usual builds. It should be activated explicitly using execution id.